apps/ledger-live-desktop/tests/models/SendModal.ts

```diff
@@ -1,11 +1,26 @@
-import { Page } from "@playwright/test";
+import { Locator, Page } from "@playwright/test";
 import { Modal } from "./Modal";
 
 export class SendModal extends Modal {
   readonly page: Page;
 
+  readonly drowdownAccount: Locator;
+  readonly recipientInput: Locator;
+
   constructor(page: Page) {
     super(page);
     this.page = page;
+    this.drowdownAccount = this.page.locator('[data-test-id="modal-content"] svg').nth(1);
+    this.recipientInput = this.page.getByPlaceholder("Enter");
```

Comment:
Is it retrieved the same way on other tests ? If it's the case then ignore this comment, else maybe use another selector

Reply:
I can try to find a more specific data-test-id I was just using debugger without thinking